libkbfs/config_local.go

```diff
@@ -935,26 +978,16 @@ func (c *ConfigLocal) EnableJournaling(
 		return err
 	}
 
-	const backpressureMinThreshold = 0.5
-	const backpressureMaxThreshold = 0.95
-	// Cap journal usage to a quarter of the free space.
-	const journalByteLimitFrac = 0.25
-	// Set the absolute journal byte limit to 50 GiB for now.
-	const journalByteLimit int64 = 50 * 1024 * 1024 * 1024
-	// Set the absolute journal file limit to 1.5 million for now.
-	const journalFileLimit int64 = 1500000
-	bdl, err := newBackpressureDiskLimiter(
-		log, backpressureMinThreshold, backpressureMaxThreshold,
-		journalByteLimitFrac, journalByteLimit, journalFileLimit,
-		defaultDiskLimitMaxDelay, journalRoot)
-	if err != nil {
-		return err
+	if limiter == nil {
```

Comment:
Can you instead make `Config` expose `c.GetDiskLimiter()` and use that here instead of passing it in? Then you'd only have to create the limiter once when constructing the config.